bin/provision.sh

```diff
@@ -19,10 +19,14 @@ if [ "$#" -ne 2 ]; then
     exit 1
 fi
 
-if [ ! -z ${WPT_CONFIG} ]; then
-    if [ ! -f ${PWD}/config/prod/sync.ini ]; then
-        echo Please add a sync.ini file to ${PWD}/config/prod
+# Config file will be copied from `sync_prod.ini` and updated by default.
+# To disable it set WPT_CONFIG=false
+if [[ ${WPT_CONFIG} != "false" ]]; then
```

Comment:
From what I've understood, the idea before was, that when we wanted to update config during deployment we should have manually copied the file to the `prod` folder and the run the provision script with env variable `WPT_CONFIG`. But recently I've changed it in ansible script to set this variable to `true` by default: https://github.com/mozilla/wpt-sync/pull/1847/files, meaning updating config by default. So I guess it also makes sense in provision script to copy the file by default and only if `WPT_CONFIG = false` do not do this (since it also means do not update it on the server).